docker-compose.yml

```diff
@@ -217,7 +217,7 @@ services:
       - .charon/cluster/node3/validator_keys:/opt/charon/keys
 
   vc4-teku:
-    image: consensys/teku:${TEKU_VERSION:-23.5.0}
+    image: consensys/teku:${TEKU_VERSION:-23.8.0}
```

Comment:
let's not update this now as teku VC from version 23.6.0 is not compatible with lighthouse (It expects those p2p spec constants).